CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 Breaking changes:
 
+- Updated `example-browser` and `example-electron` applications to remove extensions which are instead contributed by VS Code builtin extensions [#6883](https://github.com/eclipse-theia/theia/pull/6883)
```

Comment:
We should be clear that skipped extensions are deprecated and will be eventually removed, if someone wants to use them they have to be ready to maintain them in separate repos.

Reply:
> We should be clear that skipped extensions are deprecated and will be eventually removed, if someone wants to use them they have to be ready to maintain them in separate repos.

Thanks! I added an entry even if I was planning on doing so for each extension I remove after the pull request is landed.